### PR TITLE
This week in os 20

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Created file, updated config.

### CODE REVIEW

File: src/twios/2023-03-02.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+## Unknown
+
+- [calcom/cal.com] [#7212](https://github.com/calcom/cal.com/pull/7212) improve
+  typing by removing as html element ([@oscard0m])
+- [getbem/getbem.github.io]
+  [#303](https://github.com/getbem/getbem.github.io/pull/303) add anchor links
+  to headings ([@oscard0m])
+- [oscard0m/octoherd-script-review-pull-requests]
+  [#1](https://github.com/oscard0m/octoherd-script-review-pull-requests/pull/1)
+  feat: Initial version ([@oscard0m])
```
Comment (oscard0m) [inline src/twios/2023-03-02.md]: category: Octoherd

File: src/twios/2023-03-02.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  [#20333](https://github.com/renovatebot/renovate/pull/20333)
+  chore(VersioningApi): add doc comments ([@Turbo87])
+
+## Unknown
+
+- [calcom/cal.com] [#7212](https://github.com/calcom/cal.com/pull/7212) improve
+  typing by removing as html element ([@oscard0m])
+- [getbem/getbem.github.io]
+  [#303](https://github.com/getbem/getbem.github.io/pull/303) add anchor links
+  to headings ([@oscard0m])
```
Comment (oscard0m) [inline src/twios/2023-03-02.md]: Category: [BEM](https://getbem.com/)

File: src/twios/2023-03-02.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  [#20355](https://github.com/renovatebot/renovate/pull/20355) fix(cargo): fix
+  pinning for wildcard constraints ([@Turbo87])
+- [renovatebot/renovate]
+  [#20333](https://github.com/renovatebot/renovate/pull/20333)
+  chore(VersioningApi): add doc comments ([@Turbo87])
+
+## Unknown
+
+- [calcom/cal.com] [#7212](https://github.com/calcom/cal.com/pull/7212) improve
+  typing by removing as html element ([@oscard0m])
```
Comment (oscard0m) [inline src/twios/2023-03-02.md]: Category: [Cal.com](https://cal.com/)

File: src/twios/2023-03-02.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+- [embroider-build/embroider]
+  [#1359](https://github.com/embroider-build/embroider/pull/1359) Ignore resolve
+  requests that start with ! ([@mansona])
+
+## Octokit
+
+- [octokit/.github] [#38](https://github.com/octokit/.github/pull/38)
+  feat(renovate-config): automerge lockFileMaintenance changes ([@oscard0m])
+
+## TypeScript
```
Comment (oscard0m) [inline src/twios/2023-03-02.md]: @Turbo87 should this category be **Renovate** instead?

```suggestion
## Renovate
```

File: config/this-week-in-open-source.config.json [hunk 5]
```diff
@@ -76,7 +76,9 @@
                 "AeroRust/nmea",
                 "Swatinem/rust-cache",
                 "pietroalbini/rfcs",
-                "rust-lang/hashbrown"
+                "rust-lang/hashbrown",
+                "marcoow/rust_rest",
```
Comment (Mikek2252) [inline config/this-week-in-open-source.config.json:81]: @marcoow should this one be here? i'm guessing it is just a test project

File: src/twios/2023-03-02.md (hunk 4 above)
Comment (Turbo87) [reply src/twios/2023-03-02.md]: Yeah, that might make sense

Comment (emmasofiah2o) [reply src/twios/2023-03-02.md]: Is this specific to these links or in general? Have you checked the config file?

Comment (oscard0m) [reply src/twios/2023-03-02.md]: I opened a PR to change it [here](https://github.com/mainmatter/mainmatter.com/pull/2003)

**Comment by emmasofiah2o** [comment]
@oscard0m how do I pull the changes from your PR to this one? I have merged yours, but I am not sure if by rebasing this branch to master, I will undo changes?

**Comment by oscard0m** [comment]
> @oscard0m how do I pull the changes from your PR to this one? I have merged yours, but I am not sure if by rebasing this branch to master, I will undo changes?

I did that for you: https://github.com/mainmatter/mainmatter.com/compare/ea169283012737ab5bc1355aa24734a0f98726d7..45b3097c282cc5ddcb77149028423209ce49c7c8

**Comment by emmasofiah2o** [comment]
@BlueCutOfficial your PRs are in the category "unknown" for now - where do they belong? I am guessing Ember, but I am not sure :)

**Comment by BlueCutOfficial** [comment]
@emmasofiah2o yes indeed, `ember-slugify` is an Ember addon 👍

**Pull request merged**